.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: ci
+
+on: [push, pull_request, release]
```

Comment:
@halvardssm please open a separate PR for CI update - try to keep PRs focused on single problem

Reply:
@bartlomieju Sure, added one here #109